tests/cases/fourslash/quickInfoPropertyTag.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference path='fourslash.ts' />
+
+// @allowJs: true
+
+// @Filename: /a.js
+/////**
+//// * @typedef I
+//// * @property {number} x Doc
+//// */
+////
+/////** @type {I} */
+////const obj = { /**/x: 10 };
```

Comment:
what about `obj.x/**/` ? Did that work before this change? Does it work now?

Reply:
Yes, these both get the same symbol, and the documentation-getting code just depends on the symbol.